TACC/Core-Portal
# FP-1196: Push Keys Modal from 400 on Files Listing for non-public systems

## Overview: ##
Changed 502 error message for non-private systems to the generic message displayed for 400 errors. 'Push keys' 502 error message retained for private file systems.

## Related Jira tickets: ##

* [FP-1196](https://jira.tacc.utexas.edu/browse/FP-1196)

## Summary of Changes: ##
- Moved GenericMessage to top of error if/else tree in DataFilesTablePlaceholder for use with both 400 and 502 error cases.
- Updated unit test for error messages to test for both public and private 502 messages.

## Testing Steps: ##
1. Navigate to 'Data Files' -> 'Public Data'.
2. Verify the message 'An error occurred loading this directory. For help, please submit a ticket.' is shown.

## UI Photos:
![image](https://user-images.githubusercontent.com/43251554/134421184-5618bdf0-811a-4de1-ac44-fc9f13d41da1.png)

## Notes: ##
